_config.yml

```diff
@@ -0,0 +1,10 @@
+plugins:
+  - jekyll-relative-links
+relative_links:
+  enabled: true
+  collections: true
+include:
```

Comment:
At some point, please add a comment here explaining why these things were included.

Reply:
Yep, great call. And I’ll try to do better than I stole it from a tutorial…